packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -290,7 +318,7 @@ abstract class RenderToggleable extends RenderConstrainedBox {
     config.isEnabled = isInteractive;
     if (isInteractive)
       config.onTap = _handleTap;
-    config.isChecked = _value;
+    config.isChecked = _value != false;
```

Comment:
we need to convey the null state to the accessibility system

Reply:
I'd like to defer that to a separate PR. I've discussed it with @goderbauer; support for tristates isn't trivial.

Reply:
Can you leave a TODO here and/or file a bug?

Reply:
In general we should have the rule that all new widgets support accessibility, text scaling, localization, and bidi out of the box, but if there's a reason to make an exception here then ok.

Reply:
Added a TODO and filed https://github.com/flutter/flutter/issues/14677